examples_utils/benchmarks/command_utils.py

```diff
@@ -120,6 +120,33 @@ def get_benchmark_variants(benchmark_name: str, benchmark_dict: dict) -> list:
     return variants
 
 
+def remove_wandb_args(cmd: str) -> str:
+    """Remove all wandb args and their values from command strings.
+    """
+
+    new_arg_list = []
+    arg_list = cmd.split(" ")
+    
+    skip = False
+    for i in len(arg_list):
+        if "--" in arg_list[i] and "wandb" in arg_list[i]:
+            # Also remove the values given to wandb args using a skip bool for
+            # next iteration
+            if not "--" in arg_list[i+1]:
```

Comment:
it may be possible to have short flags “-i” as the next value in the arguments

Reply:
maybe something like
`if not re.match("^[-]+", str):`

Reply:
```
arg_pattern = re.compile("^[-]+")
if arg_pattern.match(arg_list[i]) and "wandb" in arg_list[i]:
    if not arg_pattern.match("^[-]+", arg_list[i+1]):
```

Reply:
Is this really cleaner? 😂 

Thanks, I'll use this

Reply:
haha its the first thing that came to my head, i think using what you had before is simpler though, [:2], taking the first two chars of the arg